roadie-kubernetes-cluster-access/values.yaml

```diff
@@ -2,7 +2,7 @@
 broker:
   enabled: false
   image: roadiehq/broker:kubernetes
-  imagePullSecret:
+  imagePullSecrets:
```

Comment:
I think singular makes sense here, given its use in the template. However maybe you want to actually provide the ability to specify the array? Im easy with either.